### DISCUSSION TITLE
Can't build extension

### DISCUSSION THREAD

**Discussion by John93-Blynk** [shreyashsaitwal/rush-cli#59] [General]
Hello,
I'm facing an error and can't build extensions at all. Can you please tell me what should I do?
![IMG_20240315_201941](https://github.com/shreyashsaitwal/rush-cli/assets/78309628/ec2c4223-bb39-451a-8bef-576b6c628bca)

**Comment by MahmoudHooda2019** [comment]
you need to be in the path of the project, like C:\Users\Aemo\extension-name or you can use you idea that is more easier
